keyboards/icebreaker/readme.md

```diff
@@ -0,0 +1,26 @@
+# Icebreaker
+
+![Icebreaker](https://i.imgur.com/vSRfEOD.png)
+
+
+
+* Keyboard Maintainer: [Matthijs Muller](https://github.com/SmollChungus)
+* Hardware Supported: Icebreaker Hotswap PCB
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ccebreaker:default
```

Comment:
```suggestion
    make icebreaker:default
```
Everything under `hotswap/` should be moved up unless you plan a non-hotswap version.